tests/monitoring_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"crypto/tls"
 	"errors"
 	"fmt"
+	"k8s.io/utils/pointer"
+	kubevirtv1 "kubevirt.io/api/core/v1"
```

Comment:
Can you move these lines to the second last block?

Reply:
Sure!